scenarios/getting_started/c/main.c

```diff
@@ -58,6 +58,7 @@ int main(int argc, char* argv[])
 
   mqtt_client_obj obj;
```

Comment:
Nit instead of below:
```c
mqtt_client_obj obj = { 0 };
```

Reply:
Nit^2: All objects that can need initialization should use this technique to avoid random values when NULL is expected, etc.